index.html

```diff
@@ -15,102 +15,104 @@
         </ul>
     </nav>
     <h1>Your Website is an investment in the<br> success of your business.</h1>
-    <h4>We are here to help you succeed.</h4>
-    <button action="https://www.google.com/search" method="get" class="button-1" type="submit">Let's Work Together</button>
+    <p>We are here to help you succeed.</p>
+    <button class="button-1" type="submit">Let's Work Together</button><br>
+    <img src="assets/icons/squiggle_line_stripe.svg" alt="icon" widht="10px"/>
 </header>
 <main>
   <section class="sec-1">
```

Comment:
Je hebt de secties van korte namen voorzien. Dit werkt natuurlijk goed voor jezelf maar als jij je CSS deelt en degene moet dit uitlezen wat is dan sec-1?

Ik hanteer altijd benaming die overeenkomt met de content die erin staat. Deze sectie gaat over Rekupe dus benoem ik deze ook als about-us. Zo loop je de gehele pagina door en voordat je start met je ccs deel je alles in blokken met de juiste benaming.

Je had bv sec-1-about-us kunnen gebruiken.